cargo-nextest/src/dispatch.rs

```diff
@@ -1006,7 +1007,35 @@ impl BaseApp {
         // Next, read the build platforms.
         let build_platforms = match reuse_build.binaries_metadata() {
             Some(kind) => kind.binary_list.rust_build_meta.build_platforms.clone(),
-            None => discover_build_platforms(&cargo_configs, cargo_opts.target.as_deref())?,
+            None => {
+                let target_triple =
+                    discover_target_triple(&cargo_configs, cargo_opts.target.as_deref());
+                let mut args = vec!["--print", "sysroot"];
+                if let Some(target_triple) = &target_triple {
+                    args.extend(["--target", target_triple.platform.triple_str()]);
```

Comment:
Ahh so actually I realized reviewing this again -- we need to obtain the target libdirs for:

1. The host, unconditionally
2. The target, if present

So we have to call rustc up to twice. (And if we support multiple `--target` in the future, N+1 times.)

I think we'll have to change this code to achieve that effect.

Reply:
Done. Detect the host and target libdirs separately.